src/assets/js/merkle_graph_d3.js

```diff
@@ -25,26 +25,35 @@ export function load_merkle_graph_js(tree_data){
 
     let merkleContainer = document.getElementById("merkle-container");
     let width = merkleContainer.clientWidth;
-    let height = width / 2
+    let height = width < 1000 ? width : width / 2
```

Comment:
e3cbd832e5ef85e5a3a221ce870f60ff5ab51b44: This seems like a magic number to me. Is there no flexible solution?